tpchgen/src/generators.rs

```diff
@@ -293,7 +293,7 @@ impl fmt::Display for Part<'_> {
     fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
         write!(
             f,
-            "{}|{}|{}|{}|{}|{}|{}|{:.2}|{}|",
+            "{}|{}|Manufacturer#{}|Brand#{}|{}|{}|{}|{:.2}|{}|",
```

Comment:
moved the formatting into Display rather than creating a String to then copy again into output